tests/framework/db/mssql/MssqlCommandTest.php

```diff
@@ -127,4 +127,27 @@ public function testAlterTable()
             ['id' => 2, 'bar' => 'hello'],
         ], $records);
     }
+
+    public function testRenameTable()
+    {
+        $db = $this->getConnection();
+
+        $fromTableName = 'type';
+        $toTableName = 'new_type';
+
+        // on the first run the 'new_type' table does not exist
+        try {
```

Comment:
This approach would work in MySQL well. Let's use it instead of duplicating tests.